pkg/core/interop/contract/call.go

```diff
@@ -69,7 +69,7 @@ func Call(ic *interop.Context) error {
 		return fmt.Errorf("method not found: %s/%d", method, len(args))
 	}
 	hasReturn := md.ReturnType != smartcontract.VoidType
-	var cb vm.ContextUnloadCallback
+	var cb func(stack *vm.Stack)
```

Comment:
Do we need a callback to decide on `Null` pushing and do it?

Reply:
We don't it can safely be replased with boolean value, fixed.